docs/content/concepts/assets/asset-auto-execution.mdx

```diff
@@ -149,52 +194,61 @@ def asset1():
+
+Setting a lazy auto-materialize policy on an asset allows it to be auto-materialized to help downstream assets meet their freshness policies. In this example, both `asset2` and `asset3` will be auto-materialized up to once per day, to help `asset3` meet its freshness policy. Conversely, if `asset2`, did not have an auto-materialize policy, then `asset3` would never become fresh, unless `asset2` were materialized in some other way.
 
-<PyObject object="StaticPartitionsDefinition" pluralize /> and <PyObject
-object="DynamicPartitionsDefinition"
-pluralize
-/> do not have this limit, so all partitions will be automatically-materialized.
+```python file=/concepts/assets/auto_materialize_lazy_transitive.py
+from dagster import AutoMaterializePolicy, FreshnessPolicy, asset
 
-## Rules of auto-materialization
 
-- Assets will not be auto-materialized if any of their ancestors are currently being auto-materialized.
-- Assets will not be auto-materialized if any of their ancestors have not yet incorporated the newest materialization of their upstream assets.
-- If the run to auto-materialize an asset fails, it can be retried if [run retries](/deployment/run-retries) are configured. Otherwise, Dagster won't try to auto-materialize that asset again until it would auto-materialize it if the failed run had succeeded. I.e., if an asset has a daily freshness policy, and it fails, Dagster won't auto-materialize it again until the next day.
-- By default, no more than one materialization of a given asset will be kicked off per minute. Further materialization requests will be discarded, and will require manual backfilling to complete. This can be configured using the `max_materializations_per_minute` argument to <PyObject object="AutoMaterializePolicy" method="eager" /> and <PyObject object="AutoMaterializePolicy" method="lazy" />.
+@asset
+def asset1():
+    ...
 
-## Run tags
 
-Runs triggered by auto-materialize policies are tagged with `dagster/auto_materialize`: `true`. Additional tags can be configured in [`dagster.yaml` (OSS)](/deployment/dagster-instance#auto-materialize) or [deployment settings (Cloud)](/dagster-cloud/managing-deployments/deployment-settings-reference#auto-materialize).
+@asset(auto_materialize_policy=AutoMaterializePolicy.lazy(), deps=[asset1])
+def asset2():
+    ...
+
+
+@asset(
+    auto_materialize_policy=AutoMaterializePolicy.lazy(),
+    freshness_policy=FreshnessPolicy(maximum_lag_minutes=24 * 60),
+    deps=[asset2],
+)
+def asset3():
+    ...
+```
+
+If multiple assets with freshness policies depend on the same upstream asset, Dagster will try to intelligently materialize the upstream asset at times that allow it to minimize the number of runs of the upstream asset, while meeting the downstream freshness policies.
```

Comment:
```suggestion
If multiple assets with freshness policies depend on the same upstream asset, Dagster will try to intelligently materialize the upstream asset at times that allow it to minimize the number of runs of the upstream asset while meeting the downstream freshness policies.
```

- Removing an unneeded comma
- What does `intelligently` mean here?

Reply:
just decided to remove "intelligently" as it doesn't really add anything to the sentence